app/scripts/modules/core/src/artifact/imageSourceSelector.component.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Please omit this. The build leaves comments in the generated app (or at least it did in the past), which we try to keep as small as possible.